app/konnect/api-products/productize-service.md

```diff
@@ -21,9 +23,28 @@ You can set up an API product and API product version by clicking {% konnect_ico
 1. Create a new name for your API product, and enter an optional **Description** and any **labels** that you want to associate with the product, then press **create**. 
 
 You will be greeted by the dashboard for the API product that you just created. You can use this dashboard to manage an API product. You can read more about this dashboard on the API products [overview page](/konnect/api-products/)
+{% endnavtab %}
+{% navtab API%}
+
+1. Create a new API product by issuing a `POST` request to the [`/api-products`](/konnect/api/api-products/latest/#/API%20Products/create-api-product) endpoint. 
```

Comment:
```suggestion
Create a new API product by issuing a `POST` request to the [`/api-products`](/konnect/api/api-products/latest/#/API%20Products/create-api-product) endpoint. 
```

I think you could probably remove the ordered list here since it's just one step